src/pudl/io_managers.py

```diff
@@ -341,6 +341,69 @@ def handle_output(self, context: OutputContext, obj: pd.DataFrame | str):
                 "queries."
             )
 
+    def load_input(self, context: InputContext) -> pd.DataFrame:
+        """Load a dataframe from a sqlite database.
+
+        Args:
+            context: dagster keyword that provides access output information like asset
+                name.
+        """
+        table_name = self._get_table_name(context)
+        _ = self._get_sqlalchemy_table(table_name)
```

Comment:
Could you add a comment indicating whether this assignment to `_` is mutating something in the class or just checking whether an exception is raised? (I think the latter, right?)